news/4.34/platform.html

```diff
@@ -181,6 +181,29 @@ <h2>General Updates </h2>
         </p>
       </td>
     </tr>
+    
+    <tr id="edge-browser-improvements"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+      <td class="title">Several Improvements made to the Edge Browser (Windows only)</td>
+      <td class="content">
+        <p>
+          In an effort to make it the default browser for Windows, several issues that affected the Edge browser 
+          have been addressed in this release.
```

Comment:
Maybe mention some of the most relevant features, like using local resources, which, e.g., makes the Welcome page usable?

Reply:
I added some of the improvements